src/Symfony/Component/Validator/Constraints/Bic.php

```diff
@@ -28,6 +30,7 @@ class Bic extends Constraint
     const INVALID_BANK_CODE_ERROR = '00559357-6170-4f29-aebd-d19330aa19cf';
     const INVALID_COUNTRY_CODE_ERROR = '1ce76f8d-3c1f-451c-9e62-fe9c3ed486ae';
     const INVALID_CASE_ERROR = '11884038-3312-4ae5-9d04-699f782130c7';
+    const INVALID_IBAN_COUNTRY_CODE_ERROR = '29a2c3bb-587b-4996-b6f5-53081364cea5';
```

Comment:
`public const`, but feel free to ignore. This is a CS discussion also (we added public for new consts before though)

Reply:
let's keep consistency